__tests__/components/graph-view.test.js

```diff
@@ -48,6 +49,7 @@ describe('GraphView component', () => {
     onUpdateNode = jest.fn();
     onSwapEdge = jest.fn();
     onSelectEdge = jest.fn();
+    onUndo = jest.fn();
```

Comment:
added onUndo to test the keyDown handler because it's a simple case (doesn't depend on selection state)